### PR TITLE
Misc: Codacy warnings cleanup.

### PR DESCRIPTION
**Pull request by lightningterror**
### Description of Changes

Misc: Codacy warnings cleanup.
### Rationale behind Changes

Less warnings.
### Suggested Testing Steps

Compiles.

### CODE REVIEW

File: pcsx2/windows/VCprojects/IopSif.cpp [hunk 1]
```diff
@@ -90,12 +90,6 @@ void FinalizeEERead()
 	CPU_INT(DMAC_SIF0, 16);
 }
 
-s32 DoSIFWrite(u32 iopAvailable)
-{
-	u32 eeAvailable = PrepareEEWrite();
```
Comment (Warepire) [inline pcsx2/windows/VCprojects/IopSif.cpp:95]: This is undefined but probably returning eeAvailable on accident. So the code in sif0DmaRead was probably doing what it was intended to do. Or is the code dead for other reasons?

Comment (lightningterror) [reply pcsx2/windows/VCprojects/IopSif.cpp:95]: The whole file isn't used, I'll be purging it.

Comment (tellowkrinkle) [reply pcsx2/windows/VCprojects/IopSif.cpp:95]: Why bother making modifications to a file you're about to delete?

Maybe just delete it now

Comment (lightningterror) [reply pcsx2/windows/VCprojects/IopSif.cpp:95]: That's what I said?

**Pull request merged**